internal/cli/atlas/serverless/create.go

```diff
@@ -61,12 +63,12 @@ func (opts *CreateOpts) Run() error {
 	return opts.Print(r)
 }
 
-func (opts *CreateOpts) newServerlessCreateRequestParams() *atlas.ServerlessCreateRequestParams {
-	return &atlas.ServerlessCreateRequestParams{
+func (opts *CreateOpts) newServerlessCreateRequestParams() *atlasv2.ServerlessInstanceDescriptionCreate {
+	return &atlasv2.ServerlessInstanceDescriptionCreate{
 		Name: opts.instanceName,
-		ProviderSettings: &atlas.ServerlessProviderSettings{
+		ProviderSettings: atlasv2.ServerlessProviderSettings{
 			BackingProviderName: opts.provider,
-			ProviderName:        providerName,
+			ProviderName:        pointer.Get(providerName),
```

Comment:
let's save the pointer util for literals
```suggestion
			ProviderName:        &providerName,
```